src/Passes/examples/Makefile

```diff
@@ -1,7 +1,15 @@
 all:
+	cd SimpleLoop && make all && make clean
 	cd TeleportChain && make all && make clean
 	cd LoopRecursion && make all && make clean
+	cd RandomWalkPE1 && make all && make clean
+	cd RandomWalkPE2 && make all && make clean
+	cd RandomWalkPE3 && make all && make clean
 
 clean:
-	cd TeleportChain && make clean
+	cd SimpleLoop && make clean	
+	cd TeleportChain && make clean	
 	cd LoopRecursion && make clean
+	cd RandomWalkPE1 && make clean
+	cd RandomWalkPE2 && make clean
```

Comment:
looks like this may need a merge from the feature branch after #1138 merges since some of the examples have changed.